src/test/unit/utils.js

```diff
@@ -134,10 +140,12 @@ function prepareUnitTestModule(args) {
     }
 
     args.options.postInit = function (wym) {
-        // TODO: We should not load all these plugins by default.
-        wym.listPlugin = new ListPlugin({}, wym);
-        wym.tableEditor = wym.table();
-        wym.structuredHeadings();
+        if (args.loadDefaultPlugins) {
+            // TODO: We should not load all these plugins by default.
```

Comment:
I'd say that the listPlugin and tableEditor should be loaded in >90% of use cases and we need to make sure they're test. In a perfect world, we would have tests both with them and without them, but that seems like a lot of extra tests.

`structuredHeadings` is a bit of a special case, at least right now.